installers/install.ps1

```diff
@@ -369,6 +369,13 @@ function install()
     }
     Move-Item (Join-Path $tmpParentPath $stateexe) $installPath
 
+    # Write install file
+    $StatePath = Join-Path -Path $installDir -ChildPath $script:STATEEXE
+    $Command = "$StatePath export config --filter=dir"
+    $ConfigDir = & Invoke-Expression $Command | Out-String
+    $InstallFilePath = Join-Path -Path $ConfigDir.Trim() -ChildPath "install.txt"
```

Comment:
Please name this `installsource.txt` or `installer.txt`, `install.txt` is too vague.